### ISSUE TITLE
Virus detected at ruffle-nightly-2024_03_11-web-selfhosted.zip

### ISSUE THREAD

**Issue by Stefannag** [ruffle-rs/ruffle#15521]
![image](https://github.com/ruffle-rs/ruffle/assets/44028084/85b39375-e13a-4eb0-bdfb-196d166c5452)

I've tried to download the last version of Ruffle Self Hosted, but it trigged the antivirus. I've downloaded multiple version on the same laptop, nothing changed in the time. 

Windows 11 with Windows Security
Version 122.0.6261.112 (Official Build) (64-bit)

**Comment by Lord-McSweeney** [comment]
This is a known issue. It happens often and is probably at least somewhat related to new packages being published nightly.

**Comment by Sunsper** [comment]
VirusTotal (ClamAV and Google engines) detect Win.Trojan.Virut-226 on Nighly releases since 07/02/2024 up to the latest at the time of writing 07/04/2024, specifically x86_32 for Windows binary. Screenshot below.
![зображення](https://github.com/ruffle-rs/ruffle/assets/75447034/54f0cd2b-7754-480a-a6c8-199dd884a6a8)

I assume this is a false positive.

I'm reporting this only because I uploaded that .zip archive to my Google Drive, and it got flagged for abuse.

**Comment by torokati44** [comment]
Thanks for reporting this! I've sent an email to VirusTotal about it - no idea whether anyone will ever look at it though.

**Comment by torokati44** [comment]
So we could report this directly to ClamAV at https://www.clamav.net/reports/fp, or just close this, deeming the as an inconsequential, intermittent glitch.

**Comment by Sunsper** [comment]
> I got a reply from VirusTotal:
> 
> > VirusTotal only aggregates data from a variety of vendors. We produce no verdicts of our own and as such, we can’t modify these results. We are not intended to be an authoritative reputation engine, but rather provide intelligence and context to users so that they can make the best decision. 1/60 and even 5/60 doesn’t automatically mean “Bad”, and 0/60 doesn't always mean good. Each decision on whether something is malicious is ultimately the responsibility of users or the security vendors who use the data to improve their services.
> > If the false positive is for a File or a URL please try re-scanning it first, as that may solve the issue.
> > If the false positive persists please reach out to the vendor that is producing it providing a link to the VirusTotal report. List of vendor contacts can be found at https://docs.virustotal.com/docs/false-positive-contacts
> > You can find a list of false positives contacts in the following link:
> > https://docs.virustotal.com/docs/false-positive-contacts

Exactly. This is the reason why I mentioned false positive at my very first comment here.

> So we could report this directly to ClamAV at https://www.clamav.net/reports/fp

In that case should be also reported to Google as well.
